engine/src/main/java/org/hibernate/validator/internal/engine/ValidatorFactoryScopedContext.java

```diff
@@ -35,6 +36,11 @@
 	 */
 	private final ExecutableParameterNameProvider parameterNameProvider;
 
+	/**
+	 *  Docs
```

Comment:
Looks simple enough to write the doc instead of having a TODO :). Wouldn't want to forget about this.